omnigibson/simulator.py

```diff
@@ -489,53 +594,36 @@ def import_scene(self, scene):
             assert self.is_stopped(), "Simulator must be stopped while importing a scene!"
             assert isinstance(scene, Scene), "import_scene can only be called with Scene"
 
-            # Clear the existing scene if any
-            self.clear()
-
-            # Initialize all global updatable object states
-            for state in self.object_state_types_requiring_update:
-                if issubclass(state, GlobalUpdateStateMixin):
-                    state.global_initialize()
+            # Check that the scene is not already imported
+            if scene in self._scenes:
+                raise ValueError("Scene is already imported!")
 
-            self._scene = scene
-            self._scene.load()
+            # Load the scene.
+            self._scenes.append(scene)
+            # TODO(parallel-hang): Actually implement this API
+            self._last_scene_edge = scene.load(idx=len(self.scenes), last_scene_edge=self._last_scene_edge)
 
             # Make sure simulator is not running, then start it so that we can initialize the scene
             assert self.is_stopped(), "Simulator must be stopped after importing a scene!"
             self.play()
 
             # Initialize the scene
-            self._scene.initialize()
+            scene.initialize()
```

Comment:
would it also be more efficient to have a `batch_import_scene()` call to avoid N stop / plays + initialization calls as we import scenes? Not sure if that would give us any perf benefits

Reply:
@cgokmen what do you think?

Reply:
I was under the impression that I had done this but I must not have merged it yet. It has happened in our RL branch. I'm hesitant to merge anything into this branch but I will create a PR for everything else that's ready immediately afterwards - this will be included there. lets close for now.